src/odemis/acq/test/stream_test.py

```diff
@@ -2903,9 +2904,9 @@ def test_arpol(self):
         """Test StaticARPOLStream"""
 
         metadata = []
-        pol_positions = list(POL_POSITIONS)
-        qwp_positions = [0.0, 1.570796, 0.785398, 2.356194, 0.0, 0.0]
-        linpol_positions = [0.0, 1.570796, 0.785398, 2.356194, 0.785398, 2.356194]
+        pol_positions = list(set(POL_POSITIONS) | {MD_POL_NONE})
```

Comment:
A set doesn't guarantee the order. Should probably be `[MD_POL_NONE] + POL_POSITIONS` to fit the other two variables.